website_signup_legal_page_required/__manifest__.py

```diff
@@ -1,17 +1,16 @@
-# -*- coding: utf-8 -*-
 # Copyright 2015 Tecnativa
 # Copyright 2016 Alessio Gerace - Agile Business Group
 # Copyright 2018 Lorenzo Battistini - Agile Business Group
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {
     "name": "Require accepting legal terms",
-    "summary": "Force the user to accept the legal terms to open an account",
-    "version": "10.0.1.0.0",
+    "summary": "Force the user to accept the legal terms to open an account.",
+    "version": "12.0.1.0.0",
     "category": "Website",
     "website": "https://github.com/OCA/website",
     "author": "Antiun Ingeniería S.L.,"
-    " Agile Business Group, Odoo Community Association (OCA)",
+    " Agile Business Group, José Luis Sandoval Alaguna, Odoo Community Association (OCA)",
```

Comment:
@SalahAdDin please add yourself to contributors instead of original authors (this line also breaks travis), possibly using the new `readme` dir: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst